stingray/events.py

```diff
@@ -85,7 +88,8 @@ class EventList(object):
     """
     def __init__(self, time=None, energy=None, ncounts=None, mjdref=0, dt=0,
                  notes="", gti=None, pi=None, high_precision=False,
-                 mission=None, instr=None):
+                 mission=None, instr=None, header=None, detector_id=None,
+                 **other_kw):
```

Comment:
Note that several of these are not documented in the docstring, and that it's not clear what `**other_kw` can or should contain?